print_list.js

```diff
@@ -0,0 +1,34 @@
+
+class Node {
+  constructor(value, next) {
+    this.value = value;
+    this.next = next;
+  }
+
+  getNext() {
+    return this.next;
+  }
+
+  setNext(next) {
+    this.next = next;
+  }
+
+  getValue() {
+    return this.value;
+  }
+
+  setValue(value) {
+    this.value = value;
+  }
+
+  printList() {
+    let current = this;
+
+    while (current != null) {
```

Comment:
might be a little cleaner to say "while (current)"
```suggestion
while (current) {
```